components/LeagueSurvivors/LeagueSurvivors.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { cn } from '@/utils/utils';
+import { ILeagueSurvivorsProps } from './LeagueSurvivors.interface';
+import * as React from 'react';
+
+const LeagueSurvivors = React.forwardRef<
+  HTMLParagraphElement,
+  ILeagueSurvivorsProps
+>(({ className, survivors, totalPlayers, ...props }) => (
+  <p
+    data-testid="LeagueSurvivors"
+    className={cn(`LeagueSurvivors text-sm text-foreground ${className}`)}
+    {...props}
```

Comment:
What is the purpose of spreading props here?

Reply:
This was spreading the props fromthe `HTMLParagraphElement`, however, in the latest commit I'll have updated this to a `JSX.Element` and it is no longer needed.

Reply:
Updated in latest commit.